docs/parts/monorepo-support.md

```diff
@@ -0,0 +1,70 @@
+You can also see demos for each of the package managers in this [monorepo demo collection](https://stackblitz.com/@d3lm/collections/workspaces).
+
+### npm
+
+Your project repository may look like this:
+
+```
+workspace-project
+├─ packages
+│  ├─ backend
+│  │  ├─ package.json
+│  │  └─ index.js
+│  ├─ frontend
+│  │  ├─ package.json
+│  │  └─ index.js
+│  └─ common
+│     ├─ package.json
+│     └─ index.js
+└─ package.json
+```
+
+To define a workspace, add the `workspaces` field to the `package.json`:
+
+```
+"workspaces": [
+  "packages/*"
+],
```

Comment:
In this page, some of our examples of fragments from `package.json` use a final comma, and some don't. Can we harmonize it?

My vote would be for no trailing comma, because:
- it looks a bit distracting;
- if you copy this config to add it at the end of your `package.json`, you have to add a comma _before_ and remove the final comma.

It's hard to predict where users will need to add commas exactly to have valid JSON, so I'd rather let users be responsible for that. But I’m fine with always including the trailing comma if we think that's better (as long as we're consistent).